ci/bugfix-release-pr-body.md

```diff
@@ -0,0 +1,63 @@
+AUTO: merge $RELEASE_BRANCH into $SOURCE_BRANCH
+
+This PR was automatically generated in response to a push to `master`,
+and is a chance to review any changes that will be included in the release
+branch before merging.  Under most circumstances, this PR will probably be
+a formality.  However, there are a few cases where we may need to do some
+extra work to make sure `$RELEASE_BRANCH` contains what it should after the
+merge:
+
+## There is a merge conflict in this PR
+
+1. Leave a comment on this PR about the merge conflict and close the PR.
+2. In your fork, make a new `pr-resolution` branch off of `$SOURCE_BRANCH`:
+   ```shell
+   $ git checkout $SOURCE_BRANCH
+   $ git pull upstream $SOURCE_BRANCH  # Include the latest changes on the upstream master
+   $ git checkout -b pr-resolution
+   $ git merge $RELEASE_BRANCH
+   ```
+3. Resolve the conflicts locally
+4. Commit the changes to `pr-resolution`.
+5. Create a PR from `pr-resolution` into `$RELEASE_BRANCH`, and include a link
+   to the origin PR in the description.
```

Comment:
Should there be a comment here about deleting the `pr-resolution` branch once everything is good to go? For when another `pr-resolution` needs to take place in the future?

Reply:
Good idea!  Patched in c44af2a